docs/commerce/cart/cart_api.md

```diff
@@ -197,7 +197,7 @@ To combine the contents of multiple shopping carts into a target cart, use the `
 This operation is helpful when you want to consolidate items from a reorder cart and a current cart into a single order.
 
 ```php
-[[= include_file('code_samples/api/commerce/src/Command/CartCommand.php', 138, 142) =]]
-[[= include_file('code_samples/api/commerce/src/Command/CartCommand.php', 142, 149) =]]
-[[= include_file('code_samples/api/commerce/src/Command/CartCommand.php', 149, 152) =]]
+[[= include_file('code_samples/api/commerce/src/Command/CartCommand.php', 151, 154) =]]
+[[= include_file('code_samples/api/commerce/src/Command/CartCommand.php', 155, 161) =]]
+[[= include_file('code_samples/api/commerce/src/Command/CartCommand.php', 162, 164) =]]
 ```
```

Comment:
In fact, this is madness. Maybe it was initially to skip the comments. I don't know. [Comments are there in prod](https://doc.ibexa.co/en/latest/commerce/cart/cart_api/#merge-carts) and they help to read the sample.

````suggestion
[[= include_file('code_samples/api/commerce/src/Command/CartCommand.php', 151, 164) =]]
```

````